Documentation/Configuration/ImageRendering/Index.rst

```diff
@@ -0,0 +1,271 @@
+﻿.. ==================================================
+.. FOR YOUR INFORMATION
+.. --------------------------------------------------
+.. -*- coding: utf-8 -*- with BOM.
+
+.. include:: ../../Includes.txt
+
+
+===============
+Image Rendering
+===============
+
+Bootstrap Package comes with a very flexible responsive image rendering and
+supports multiple variants for various device sizes. Each variant defines the
+**break point**, **width** and the optional **sizes** for high resolution image
+support.
+
+
+
+Image Variants
+==============
+
+Bootstrap Package defines this variants by default:
+
+* default
+
+   * breakpoint: 1200px
+   * width: 1100px
```

Comment:
So, this means after the breakpoint 1200px is reached, the max image width is 1100px ? Between 992px - 1200px the max image width is 920px.

Reply:
I added this example to the docu now